track/templates/en/domains.html

```diff
@@ -3,6 +3,7 @@
 {% block title %}Check compliance - By Domain{% endblock %}
 {% block pageid_en %}domains{% endblock %}
 {% block pageid_fr %}domaines{% endblock %}
+{% block description %}This dashboard reports how federal government websites and web services are meeting good web security practices, as outlined in Information Technology Policy Implementation Notice (ITPIN): Implementing HTTPS for Secure Web Connections.{% endblock description %}
```

Comment:
Does the thing Courtney mentioned in that other PR about abbreviation tags matter her.

Reply:
Nope! These tags are what provide the description text in a google search, since it's just plain text abbrv tags would just look odd here.